<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoSimulacion</class>
 <widget class="QDialog" name="DialogoSimulacion">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>480</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1">
    <widget class="QLabel" name="label_6">
     <property name="styleSheet">
      <string notr="true">font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Procesos Bloqueados</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" rowspan="3" colspan="2">
    <widget class="QListView" name="listaLoteActual">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font-weight: bold</string>
     </property>
     <property name="text">
      <string>Proceso En Ejecuci√≥n</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">font-weight: bold</string>
     </property>
     <property name="text">
      <string>Procesos Listos</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_proceso_actual">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true">font-weight:bold</string>
     </property>
     <property name="text">
      <string>Procesos Terminados</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-weight: bold</string>
     </property>
     <property name="text">
      <string>Procesos Nuevos</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="labelNuevos">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Tiempo Actual</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Clock" name="clock">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 0, 0);
font:  &quot;Agency FB&quot;;
color:rgb(0, 255, 127);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QListView" name="listaProcesosBloqueados">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="4" rowspan="5" colspan="2">
    <widget class="QListView" name="listaProcesosTerminados">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Clock</class>
   <extends>QLabel</extends>
   <header>Visual/clock.h</header>
   <slots>
    <signal>updated(unsigned)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
